integrations/google-tag-manager/test/index.test.js

```diff
@@ -11,7 +11,8 @@ describe('Google Tag Manager', function() {
   var gtm;
   var options = {
     containerId: 'GTM-M8M29T',
-    environment: ''
+    environment: '',
+    dataLayer: ''
```

Comment:
Curious why you have to specify dataLayer as `''` vs leaving it undefined? Is this because the integration code initializes it as `''`?

Reply:
Could change this value to `dataLayerName` instead. I'm not sure there's a material difference amongst "null", "undefined", or an empty string in this integration since we're only interested in whether it's a truthy or falsy value in the code itself.

Reply:
Ooh I do like `dataLayerName` instead.. more descriptive of the type of variable.

Yeah I agree no material difference in those falsey values for integrations. Just curious why it was added in that case. No biggie.